instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/TracingFilter.java

```diff
@@ -58,8 +58,8 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
     } else {
       TraceContextOrSamplingFlags extracted = extractor.extract(invocation.getAttachments());
       span = extracted.context() != null
-          ? tracer.joinSpan(extracted.context())
```

Comment:
can you please revert the formatting change?

Reply:
sorry about that, already rollback, pls. check.